From ee57e4e09691db4f9b6e8456e5fb3bc9c13537a9 Mon Sep 17 00:00:00 2001
From: dineshsv <dineshsv@ami.com>
Date: Fri, 11 Nov 2022 13:07:15 +0530
Subject: [PATCH] Added Get BT Interface Capabilities  ipmi command in
 ipmi-whitelist.conf file

Git link:

https://git.ami.com/megarac/osp-team/osp/-/issues/2890

Tested:

  Verified-Get BT Interface Capabilities working as expected.

```shell

Before fix-Before its not added in whitelist file ,so getting response as insufficient priveledge level.

msgubuntu@Dell-server-3:~$ ipmitool -H 10.0.124.94 -U root -P 0penBmc -I lanplus -C 17 raw 0x06 0x36
Unable to send RAW command (channel=0x0 netfn=0x6 lun=0x0 cmd=0x36 rsp=0xd4): Insufficient privilege level

After fix-Added the Get BT Interface Capabilities ipmi command in whitelist file.

msgubuntu@Dell-server-3:~$ ipmitool -H 10.0.125.13 -U root -P 0penBmc -I lanplus -C 17 raw 0x06 0x36
 01 3f 3f 0a 01
msgubuntu@Dell-server-3:~$

```

Signed-off-by: dineshsv <dineshsv@ami.com>
---
 ipmi-whitelist.conf | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ipmi-whitelist.conf b/ipmi-whitelist.conf
index a20f338..47c396c 100644
--- a/ipmi-whitelist.conf
+++ b/ipmi-whitelist.conf
@@ -54,6 +54,7 @@
 0x06:0x33:0x8080   //<App>:<Get Message>
 0x06:0x34:0x7f7f   //<App>:<Send Message>
 0x06:0x35:0x8080   //<App>:<Read Event Message Buffer>
+0x06:0x36:0xff7f   //<APP>:<Get BT Interface Capabilities>
 0x06:0x37:0xff7f   //<App>:<Get System GUID>
 0x06:0x38:0xff7f   //<App>:<Get Channel Authentication Capability>
 0x06:0x39:0xff7f   //<App>:<Get Session Challenge>
-- 
2.25.1

